src/main/java/com/fryrank/model/Review.java

```diff
@@ -5,6 +5,8 @@
 import org.springframework.data.annotation.Id;
 import org.springframework.data.mongodb.core.mapping.Document;
 
+import java.util.List;
```

Comment:
Doesn't look used

Reply:
Done in [98ac0c5](https://github.com/NickPriv/FryRankBackend/pull/83/commits/98ac0c5d7f9c27e6e75a242c815fd2e47c9cbdb6).